github/github.go

```diff
@@ -490,13 +494,20 @@ These are the possible validation error codes:
         the formatting of a field is invalid
     already_exists:
         another resource has the same valid as this field
+    custom:
+        some resources return this (e.g. github.User.CreateKey()), additional
+        information is set in the Message field of the Error
 
 GitHub API docs: http://developer.github.com/v3/#client-errors
 */
 type Error struct {
 	Resource string `json:"resource"` // resource on which the error occurred
 	Field    string `json:"field"`    // field on which the error occurred
 	Code     string `json:"code"`     // validation error code
+	// For .Code == "custom" we always get a message.
+	// The github.User.CreateKey() returns such a code when trying to add a key
+	// which is already in use.
+	Message string `json:"message"`
```

Comment:
I think you can simplify the docs here. Example of when it happens is already described above, in the doc for `Error` type.

This can be just:

```Go
// Message field describing the error. Errors with Code == "custom" will always have this set.
```